LibraryApp/src/app/screens/BookList/index.js

```diff
@@ -3,7 +3,7 @@ import { View, FlatList } from 'react-native';
 import { NavigationActions } from 'react-navigation';
 import PropTypes from 'prop-types';
 import Routes from '@constants/routes';
-import { books } from '@directory/books';
+import { books } from '@rootDirectory/books';
```

Comment:
This file should be in inside `screens/BookList` so you wont need this kind of import